dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -405,4 +414,44 @@ public static long countFailures(List<ArtifactResults> table, String columnName)
         .filter(row -> row.getResult(columnName) == null || row.getFailureCount(columnName) > 0)
         .count();
   }
+
+  /**
+   * Returns mapping from jar file to simplified description of its {@link DependencyPath}s. A
```

Comment:
I think we're doing a little more than that. We're identifying the true cause of the problem.

Reply:
Enriched the javadoc.